.evergreen/config.yml

```diff
@@ -773,15 +799,29 @@ buildvariants:
     - rhel70-small
   tasks:
      - name: "test"
-- matrix_name: app-tests
+- matrix_name: app-tests-ruby-3.0
+  matrix_spec:
+    driver: current
+    ruby: ruby-3.0
+    mongodb-version: '4.4'
+    topology: standalone
+    app-tests: yes
+    # Not currently testing rails master
+    rails: ['6.0', '6.1']
+  display_name: "app tests ${driver}, ${ruby}, ${rails}"
+  run_on:
+    - ubuntu1804-small
+  tasks:
+    - name: "test"
+- matrix_name: app-tests-ruby-2.7
   matrix_spec:
     driver: current
     ruby: ruby-2.7
     mongodb-version: '4.4'
     topology: standalone
     app-tests: yes
     # Not currently testing rails master
-    rails: ['5.1', '5.2', '6.0', '6.1']
+    rails: ['5.2', '6.0', '6.1']
```

Comment:
Does this not remove test coverage for 5.1?

Reply:
Oh, good catch, thanks!

Reply:
Fixed